index.html

```diff
@@ -204,6 +204,17 @@ <h2>Securing the VC Data Model</h2>
       transformation, while at the same time supporting registered 
       claims that are understood in the context of JOSE and COSE.
     </p>
+    <p>
+      It is RECOMMENDED that media types be used to distinguish <a data-cite="VC-DATA-MODEL#credentials">verifiable credentials</a>
+      and <a data-cite="VC-DATA-MODEL#presentations">verifiable presentations</a> from other kinds of secured JSON or CBOR.
+    </p>
+    <p>
+      The most specific media type (or subtype) available SHOULD be used, instead of
+      more generic media types (or supertypes). For example, rather than the general
+      <code>application/sd-jwt</code>, <code>application/vc+ld+json+sd-jwt</code>
+      ought to be used, unless there is a more specific media type that would even
+      better identify the secured envelope format.
+    </p>
```

Comment:
```suggestion
    </p>
    <p>
    If implementations do not know which media type to use, media types defined in this specification MUST be used.
    </p>
```
not sure about MUST, but a sentence like this is needed to connect this paragraph to the rest of the SHOULDs